packages/url.lua

```diff
@@ -32,25 +125,55 @@ SILE.registerCommand("href", function (options, content)
       borderoffset = options.borderoffset },
       content)
   else
+    -- FIXME
```

Comment:
I'm not a big fan of "fixme" comments it the code as they are hard to discuss and track. Lets add any user facing documentation to the manual, any developer-only explanation as regular comments, and then open issues for any todo/fixme type comments or anything else that might generate developer discussion.

Reply:
Right. So #1239 opened, and FIXME comment removed here, and missing documentation added.